src/trigger.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <span ref="trigger" :class="[addClass || (this.trigger === 'click' ? 'click-trigger' : 'other-trigger')]"><slot></slot></span>
+    <span ref="trigger" :class="[addClass]"><slot></slot></span>
```

Comment:
Hmm, how about we do: `[(this.trigger === 'click' ? 'click-trigger' : 'other-trigger'), addClass]`, instead of entirely reverting the changes? Will that still allow MarkBind/markbind#924's implementation to work?

Reply:
No, because it will then depend on the order the classes are loaded. Since markbind.css is loaded before vue-strap.min.js, `click-tigger` will be loaded last and take precedence.

I'll try implementing MarkBind/markbind#924 by adding a `hoverable` attribute to popover instead.

Reply:
Ok sure.